R/html_dependency.R

```diff
@@ -366,11 +369,11 @@ copyDependencyToDir <- function(dependency, outputDir, mustWork = TRUE) {
   }
 
   srcfiles <- file.path(dir, files)
-  if (any(!file.exists(srcfiles))) {
+  if (any(bad <- !file.exists(srcfiles))) {
```

Comment:
I think the assignment in an `if()` can be pretty confusing, so this should be moved out. (I'll take care of this.)